### PR TITLE
Presenter: Legacy cleanup

### PR DESCRIPTION
**Pull request by gblakeman**
Removes a bunch of legacy 1.0 stuff from `Presenter` that we no longer need in the UI views. We also get a bunch of nice performance gains as a side-effect (no more image-parsing or redundant assignment metadata lookups on each selected node).

### CODE REVIEW

File: src/code.ts [hunk 1]
```diff
@@ -105,7 +98,7 @@ const dispatcher = async (action: {
         break;
       }
       case 'focus-ui': {
```
Comment (gblakeman) [inline src/code.ts:100]: This case is disabled for beta purposes, but will show up again in my next round of work, so leaving it as a placeholder for now.

File: src/Presenter.ts [hunk 2]
```diff
@@ -499,91 +448,28 @@ export default class Presenter {
...
-          || (assignment === ASSIGNMENTS.avatarNewsletter.id)
-          || (assignment === ASSIGNMENTS.avatarPerson.id)
-          || (assignment === ASSIGNMENTS.avatarSchool.id)
-        ) {
-          assignment = ASSIGNMENTS.unassigned.id as RealishAssignment;
-        }
-      } else {
+      // set “unassigned” as default
+      if (!assignment) {
         assignment = ASSIGNMENTS.unassigned.id as RealishAssignment;
```
Comment (gblakeman) [inline src/Presenter.ts:464]: The UI expects an `assignment`, so we set “unassigned” to anything we could not find an assignment on.

File: src/code.ts [hunk 3]
```diff
@@ -121,18 +114,18 @@ const dispatcher = async (action: {
         break;
       case 'show-plugin':
       case 'tools':
-        App.showToolbar(sessionKey);
```
Comment (gblakeman) [inline src/code.ts:124]: This entire `sessionKey` convention may go away, eventually, but a few legacy things still rely on it.

**Review by joeyyang — APPROVED** [review]
Looks good!

**Pull request merged**